de/privacy/current.md

```diff
@@ -15,6 +15,10 @@ Serlo.org nutzt externe Dienstleister zur Erfassung und Verarbeitung der erhoben
 
 Im Falle des Einsatzes unseres Angebots in der Schule verarbeiten wir als Auftragsverarbeiter Ihre Daten. Die verantwortliche Stelle ist Ihre Schule bzw. der Schulträger.
 
+## Laden von Drittinhalten
+
+Für Drittinhalte die nach Konsent gemäß Art. 6 Abs. 1 lit. a der DSGVO geladen werden, kann jederzeit die Berechtigung unter [serlo.org/consent](https://de.serlo.org/consent) zurückgezogen werden.
+
```

Comment:
A little bit unnecessary, since it has already this information in the final rendering of page:
![image](https://github.com/user-attachments/assets/6851ec3d-fc41-46e0-a7d2-8f574310f211)
But it is not a problem though